DanielBergshoeff/GlobalFlock.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class GlobalFlock : MonoBehaviour {
+
+    private GameObject fishPrefab;
+    private Vector3 tankSize = Vector3.zero;
+    private int numFish = 75;
+    private GameObject[] allFish;
+    private Vector3 goalPos = Vector3.zero;
+
+    
+    private void OnDrawGizmosSelected() {
+        Gizmos.color = new Color(1, 0, 0, 0.5f);
+        Gizmos.DrawCube(transform.position, new Vector3(tankSize.x * 2, tankSize.y * 2, tankSize.z * 2));
+        Gizmos.color = new Color(0, 1, 0, 1);
+        Gizmos.DrawSphere(goalPos, 0.1f);
+    }
+
+
+    // Use this for initialization
+    private void Start() {
+        GoalPos = transform.position;
+        AllFish = new GameObject[numFish];
+
+        for (int i = 0; i < numFish; i++) {
+            Vector3 pos = new Vector3(Random.Range(gameObject.transform.position.x - tankSize.x, gameObject.transform.position.x + tankSize.x),
+                                      Random.Range(gameObject.transform.position.y - tankSize.y, gameObject.transform.position.y + tankSize.y),
+                                      Random.Range(gameObject.transform.position.z - tankSize.z, gameObject.transform.position.z + tankSize.z));
```

Comment:
Ziet er netjes uit. Alleen op regel 29 en 30 is de indentie net een "tab" minder dan op regel 41 en 42.

Reply:
Ik zie het, slordig van mij, zal ik zeker aanpassen, bedankt voor je feedback!